### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by TatianaFomina**
### New
image

### CODE REVIEW

**Review by neSpecc — APPROVED** [review]
Increment a version pls

**Comment by talyguryn** [comment]
editor-js/translate-inline#1 👌

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codexteam/icons",
-  "version": "0.1.1-rc.0",
+  "version": "0.1.2",
```
Comment (talyguryn) [inline package.json:3]: let's use 0.2.0 version as new icons is a feature?

we need to apply the agreement for a versioning. we had a discussion with @neSpecc about this project.

`*.*.X` — patch for fixes 
`*.X.0` — minor for features 
`X.0.0` — major for no backwards compatibility features

**Review by talyguryn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (TatianaFomina) [reply package.json]: okay, updated the version

**Review by talyguryn — APPROVED** [review]
❤️

**Pull request merged**